src/nodes/node.h

```diff
@@ -39,7 +42,7 @@ class Node
     // Input parameters
     std::map<std::string_view, std::shared_ptr<ParameterBase>> inputs_;
     // Inbound Links
-    std::map<std::string_view, ParameterLink> links_;
+    Edge links_;
```

Comment:
Perhaps just for clarity, and assuming the typename change above:
```suggestion
    LinkMap inputLinks_;
```